politeiawww/wsdcrdata.go

```diff
@@ -42,6 +41,7 @@ type wsDcrdata struct {
 	shutdown      bool
 	client        *client.Client      // dcrdata websocket client
 	subscriptions map[string]struct{} // Client subscriptions
+	url           string
```

Comment:
Would prefer this be named `host` so that the naming convention is kept consistent throughout politeiawww.

Reply:
Actually, scratch that.  You were right here.  This is the full url.  Not just the host.